src/Release/Entity/PublicRelease.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Clearbooks\Labs\Release\Entity;
```

Comment:
@VolodymyrSergeyev I agree with this, interfaces go in the UseCase\<use case name> namespace, concrete implementations for entitys go in the entity folder